runtime/doc/lua.txt

```diff
@@ -2864,37 +2958,37 @@ tested against a version, using |vim.version.range()|.
 Supported range specs are shown in the following table. Note: suffixed
 versions (1.2.3-rc1) are not matched. >
 
-  1.2.3             is 1.2.3
-  =1.2.3            is 1.2.3
-  >1.2.3            greater than 1.2.3
-  <1.2.3            before 1.2.3
-  >=1.2.3           at least 1.2.3
-  ~1.2.3            is >=1.2.3 <1.3.0       "reasonably close to 1.2.3"
-  ^1.2.3            is >=1.2.3 <2.0.0       "compatible with 1.2.3"
-  ^0.2.3            is >=0.2.3 <0.3.0       (0.x.x is special)
-  ^0.0.1            is =0.0.1               (0.0.x is special)
-  ^1.2              is >=1.2.0 <2.0.0       (like ^1.2.0)
-  ~1.2              is >=1.2.0 <1.3.0       (like ~1.2.0)
-  ^1                is >=1.0.0 <2.0.0       "compatible with 1"
-  ~1                same                    "reasonably close to 1"
-  1.x               same
-  1.*               same
-  1                 same
-  *                 any version
-  x                 same
-
-  1.2.3 - 2.3.4     is >=1.2.3 <=2.3.4
-
-  Partial right: missing pieces treated as x (2.3 => 2.3.x).
-  1.2.3 - 2.3       is >=1.2.3 <2.4.0
-  1.2.3 - 2         is >=1.2.3 <3.0.0
-
-  Partial left: missing pieces treated as 0 (1.2 => 1.2.0).
-  1.2 - 2.3.0       is 1.2.0 - 2.3.0
-
-<
-
-cmp({v1}, {v2})                                            *vim.version.cmp()*
+   1.2.3             is 1.2.3
+   =1.2.3            is 1.2.3
```

Comment:
3 spaces seems weird and not intentional.

Reply:
I'm pretty sure it's because `version.lua` has 1 space of padding.